general_biowaste.md

```diff
@@ -2,25 +2,279 @@
 
 ## :page_facing_up: Biowaste instructions:
 
-1. Observe carefully the image to find any object of the categories. There are a lot of things in those images so take your time to find any anomaly in the image. In this project, **everyting that is not compost is an anomaly**.
+###  To segment anomalies also called intruders in biowaste, please do the following: 
 
-2. When a whole object is split in two or more parts because there's another object in front of it masking part of it, please draw both parts separately and then use the "same object" relation to indicate that both parts go together. See below how to use this tool in case there are more than 2 parts.
+1. Read carefully the description of the label listed, the section how to tag and the examples bellow.
 
-3. You have two types of labels: some with a "category" suffix, and some without. The ones with "category" are main categories, the ones without are the subcategories. Always try to find a specific tag in the subcategories. If you can't find any that fit, use one of the categories. If you can't find any that fits, use "anomaly". Please check the decision tree. 
+2. Carefully observe the image to find any object of the labels. Biowaste is made of strictly organic material from food rests or from the garden. Processed materials even if organic like paper or wood furniture are not included in the definition biowaste and are intruders. The images are dense and complex, containing objects with various shape and colors. Take your time to find any anomaly in the image. In this project, **everything that is not biowaste is an anomaly**.
 
-4. Skip all the images that have no anomalies.
+3. You have two types of labels: some with a "category" suffix, and some without. The ones with "category" are main categories, the ones without are the subcategories we will call "types". Always try to find a specific tag corresponding to a class label. If you can't find any that fit, use one of the categories. All types belong to a category as listed below.
+
+4. If you can clearly identify an object and there is not category for it, please propose a new category or just mark it.
```

Comment:
Qu'est-ce qu'on entend par "mark it"? Qu'ils nous la signalent? Dans ce cas je dirais carrément qu'il faut qu'ils nous écrivent :)